yarn.lock

```diff
@@ -658,7 +658,7 @@ center-align@^0.1.1:
     align-text "^0.1.3"
     lazy-cache "^1.0.3"
 
-chai@^4.1.2:
+chai@4.1.2:
```

Comment:
Can you explain to me what a change in these lock files actually means? Implications, reasons etc

Reply:
It appears for some reason I had a dependency not pinned down in the boilerplate. When I performed Yarn install it pinned the dependency for us

Reply:
Not in this case, but like when should it change, and when should it not change, is what I mean.

Reply:
Okay so my understanding of how Yarn changes it automatically are limited, but I believe that every time you perform an install, it looks for updates that fit within the minor release of what is installed `(x.<here>.x)`. This is what npm does. The difference is it documents these changes in a lockfile, and you can actually specify for yarn to install the exact dependencies in the lock file (as our CI will do) using `yarn install --frozen-lockfile`. I might well be wrong that this is the way it works

Reply:
https://yarnpkg.com/en/docs/configuration#toc-use-yarn-lock-to-pin-your-dependencies